### PR TITLE
feat(ui): TE-1149 add markdown support in alert create form helper tooltip

### PR DESCRIPTION
**Pull request by alcatraz627**
#### Issue(s)

[TE-1149](https://cortexdata.atlassian.net/browse/TE-1149)

#### Description

- Added markdown support in form helper tooltip in alert create form
- Using [react-markdown](https://github.com/remarkjs/react-markdown) in a common component to enable markdown support

#### Screenshots

Screenshot_9

[TE-1149]: https://cortexdata.atlassian.net/browse/TE-1149?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[TE-1149]: https://cortexdata.atlassian.net/browse/TE-1149?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: thirdeye-ui/src/app/components/parse-markdown/parse-markdown.component.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                <Typography variant="h5">{children}</Typography>
+            ),
+            h6: ({ children }) => (
+                <Typography variant="h6">{children}</Typography>
+            ),
+            a: ({ children, href, ...props }) => (
+                <LinkV1
+                    {...props}
+                    color="primary"
+                    href={`//${href}`}
```
Comment (spham92) [inline thirdeye-ui/src/app/components/parse-markdown/parse-markdown.component.tsx:53]: What's the `//` for?

Comment (alcatraz627) [reply thirdeye-ui/src/app/components/parse-markdown/parse-markdown.component.tsx:53]: The links are all opening as relative links. Even when there is an `https://` present in the link, it opens like `http://localhost:7004/alerts/create/new/simple/https://www.google.com`

Is this `//` fix fine, or should I look into using a different link component?

Comment (spham92) [reply thirdeye-ui/src/app/components/parse-markdown/parse-markdown.component.tsx:53]: Try passing `externalLink` to see if it still gives you issues like

```
<LinkV1
                    {...props}
                    externalLink
                    color="primary"
```

Comment (alcatraz627) [reply thirdeye-ui/src/app/components/parse-markdown/parse-markdown.component.tsx]: Added `externalLink`

**Review by spham92 — APPROVED** [review]
Good stuff, keeping it nice an simple

**Pull request merged**